api.py

```diff
@@ -663,15 +667,14 @@ def _ingest_entity_set(dataset_id, entity_set, **kwargs):
         raise ValueError('parameter entity_set is not an \'entities\' dict')
 
     if kwargs.get('debug'):
+        kwargs['debug'] = False
```

Comment:
What's up with this?

Is this just an optimization to lift this out of the loop?

It does have slightly different behavior than before.  If entity_set['entities'] is empty, this will run whereas it wouldn't have before.

Reply:
My intention was that this was just cleanup/optimization.  Thanks for pointing out the change in behavior, but I don't think it is relevant to execution in either mode.